noir-projects/noir-protocol-circuits/crates/private-kernel-lib/src/private_kernel_tail.nr

```diff
@@ -204,15 +103,15 @@ mod tests {
     }
 
     #[test]
-    unconstrained fn execution_succeeded() {
+    fn execution_succeeded() {
```

Comment:
Made the tests constrained so that the output validator can be ran.
Most of the tests will be removed from here and done for the output composer or the output validator (in a future pr).